spec/order_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'order'
+
+describe 'Order' do
+
+  subject(:order) { Order.new }
+  let(:menu) { double :menu }
+  let(:messager) { double :messager }
+
+  describe '#initialize' do
+    it 'should have an empty order list' do
+      expect(order.list).to be_empty
+    end
+
+    it 'should have a menu' do
+      expect(order.menu).not_to be(nil)
+    end
+
+  end
+
+  context 'customer can see menu and place order' do
+
+    describe '#read_menu' do
+      it 'displays the menu' do
+        expect{order.read_menu}.to output.to_stdout
+      end
+    end
+
+    describe '#choose' do
+      it 'allows customer to choose items and add to list' do
+         order.choose("margherita")
+         expect(order.list).to include("margherita" => 6)
+      end
+
+      it 'confirms the correct number and items to customer' do
+         expect(order.choose("margherita", 2)).to eq "You have added 2 x margherita to your order."
```

Comment:
Metrics/LineLength: Line is too long. [99/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)